Update of all Jitsi Meet components

cleeyv opened the pull request:
###### Description of changes

This updates all of the Jitsi Meet components in nixpkgs to their latest stable versions. I have done a basic test deployment of these updated versions (including jibri) and everything seemed to work well!

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

cleeyv committed:
jibri: 8.0-93 -> 8.0-139

- Fix of accidental downgrade in #158911
- Update of runtime from Java 8 -> 11

cleeyv committed:
jitsi-meet-prosody: 1.0.6600 -> 1.0.6644

cleeyv committed:
jicofo: 1.0-934 -> 1.0-940

cleeyv committed:
jitsi-videobridge: 2.2-22 -> 2.2-45

cleeyv committed:
jitsi-meet: 1.0.6600 -> 1.0.6644

ajs124 commented:
The jibri test fails for me, like this:
```
machine # [   33.100319] jicofo-start[1021]: Oct 29, 2022 3:46:19 PM org.jitsi.utils.logging2.LoggerImpl log
machine # [   33.101212] jicofo-start[1021]: INFO: Added new videobridge: Bridge[jid=jvbbrewery@internal.machine/machine, version=2.2.45-ge8b20f06, relayId=null, region=null, stress=0.00]
machine # [   33.651475] jicofo-start[1021]: Oct 29, 2022 3:46:20 PM org.jitsi.utils.logging2.LoggerImpl log
machine # [   33.652484] jicofo-start[1021]: INFO: Scheduled health-check task for: Bridge[jid=jvbbrewery@internal.machine/machine, version=2.2.45-ge8b20f06, relayId=null, region=null, stress=0.00]
machine # [   34.229497] jicofo-start[1021]: Oct 29, 2022 3:46:21 PM org.glassfish.jersey.server.wadl.WadlFeature configure
machine # [   34.230604] jicofo-start[1021]: WARNING: JAXBContext implementation could not be found. WADL feature is disabled.
machine # [   34.317279] jitsi-videobridge2-start[1020]: WARNING: [1] org.glassfish.jersey.server.wadl.WadlFeature.configure: JAXBContext implementation could not be found. WADL feature is disabled.
machine # [   34.674766] jicofo-start[1021]: Oct 29, 2022 3:46:21 PM org.glassfish.jersey.internal.inject.Providers checkProviderRuntime
machine # [   34.676023] jicofo-start[1021]: WARNING: A provider org.jitsi.rest.Version registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime. Due to constraint configuration problems the provider org.jitsi.rest.Version will be ignored.
machine # [   34.681501] jicofo-start[1021]: Oct 29, 2022 3:46:21 PM org.glassfish.jersey.internal.inject.Providers checkProviderRuntime
machine # [   34.685666] jicofo-start[1021]: WARNING: A provider org.jitsi.rest.prometheus.Prometheus registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime. Due to constraint configuration problems the provider org.jitsi.rest.prometheus.Prometheus will be ignored.
machine # [   34.993780] jitsi-videobridge2-start[1020]: WARNING: [1] org.glassfish.jersey.internal.inject.Providers.checkProviderRuntime: A provider org.jitsi.rest.Health registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime. Due to constraint configuration problems the provider org.jitsi.rest.Health will be ignored.
machine # [   34.997503] jitsi-videobridge2-start[1020]: WARNING: [1] org.glassfish.jersey.internal.inject.Providers.checkProviderRuntime: A provider org.jitsi.rest.Version registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime. Due to constraint configuration problems the provider org.jitsi.rest.Version will be ignored.
machine # [   35.007700] jitsi-videobridge2-start[1020]: WARNING: [1] org.glassfish.jersey.internal.inject.Providers.checkProviderRuntime: A provider org.jitsi.rest.prometheus.Prometheus registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime. Due to constraint configuration problems the provider org.jitsi.rest.prometheus.Prometheus will be ignored.
machine # [   36.564637] jicofo-start[1021]: Oct 29, 2022 3:46:23 PM org.eclipse.jetty.server.handler.ContextHandler doStart
machine # [   36.566216] jicofo-start[1021]: INFO: Started o.e.j.s.ServletContextHandler@16f0ec18{/,null,AVAILABLE}
machine # [   36.623792] jicofo-start[1021]: Oct 29, 2022 3:46:23 PM org.eclipse.jetty.server.AbstractConnector doStart
machine # [   36.625282] jicofo-start[1021]: INFO: Started ServerConnector@151ef57f{HTTP/1.1, (http/1.1)}{0.0.0.0:8888}
machine # [   36.701152] jicofo-start[1021]: Oct 29, 2022 3:46:23 PM org.eclipse.jetty.server.Server doStart
machine # [   36.839537] jicofo-start[1021]: INFO: Started Server@3531f3ca{STARTING}[11.0.10,sto=0] @23865ms
machine # [   36.881380] jitsi-videobridge2-start[1020]: INFO: [1] org.eclipse.jetty.server.handler.ContextHandler.doStart: Started o.e.j.s.ServletContextHandler@12a14b74{/,null,AVAILABLE}
machine # [   36.902824] jitsi-videobridge2-start[1020]: INFO: [1] org.eclipse.jetty.server.AbstractConnector.doStart: Started ServerConnector@5167268{HTTP/1.1, (http/1.1)}{127.0.0.1:8080}
machine # [   36.911809] jitsi-videobridge2-start[1020]: INFO: [1] org.eclipse.jetty.server.Server.doStart: Started Server@159e366{STARTING}[11.0.10,sto=0] @24084ms
machine # [   37.170828] prosody[1097]: c2s2dd79f0: Received[c2s]: <presence to='jvbbrewery@internal.machine/machine' xml:lang='en-US'>
machine # [   37.172715] prosody[1097]: internal.machine:muc: presence update for jvbbrewery@internal.machine/machine from session jvb@auth.machine/YFpi3EwZyPs-
machine # [   37.183702] prosody[1097]: c2s2736ef0: Sending[c2s]: <presence xml:lang='en-US' from='jvbbrewery@internal.machine/machine' to='focus@auth.machine/focus'>
machine # [   37.186594] prosody[1097]: c2s2dd79f0: Sending[c2s]: <presence xml:lang='en-US' from='jvbbrewery@internal.machine/machine' to='jvb@auth.machine/YFpi3EwZyPs-'>
machine # [   42.157785] prosody[1097]: c2s2dd79f0: Received[c2s]: <presence to='jvbbrewery@internal.machine/machine' xml:lang='en-US'>
machine # [   42.159745] prosody[1097]: internal.machine:muc: presence update for jvbbrewery@internal.machine/machine from session jvb@auth.machine/YFpi3EwZyPs-
machine # [   42.168831] prosody[1097]: c2s2736ef0: Sending[c2s]: <presence xml:lang='en-US' from='jvbbrewery@internal.machine/machine' to='focus@auth.machine/focus'>
machine # [   42.176799] prosody[1097]: c2s2dd79f0: Sending[c2s]: <presence xml:lang='en-US' from='jvbbrewery@internal.machine/machine' to='jvb@auth.machine/YFpi3EwZyPs-'>
machine # [   43.652095] prosody[1097]: c2s2736ef0: Received[c2s]: <iq id='SPSYT-13' xml:lang='en-US' type='get' to='jvbbrewery@internal.machine/machine'>
machine # [   43.653472] prosody[1097]: internal.machine:muc: focus@auth.machine/focus sent private iq stanza to jvbbrewery@internal.machine/machine (jvb@auth.machine/YFpi3EwZyPs-)
machine # [   43.655261] prosody[1097]: c2s2dd79f0: Sending[c2s]: <iq type='get' to='jvb@auth.machine/YFpi3EwZyPs-' id='anZiQGF1dGgubWFjaGluZS9ZRnBpM0V3WnlQcy0AU1BTWVQtMTMALk0hffd8QUs=' from='jvbbrewery@internal.machine/focus' xml:lang='en-US'>
machine # [   43.663962] prosody[1097]: c2s2dd79f0: Received[c2s]: <iq id='anZiQGF1dGgubWFjaGluZS9ZRnBpM0V3WnlQcy0AU1BTWVQtMTMALk0hffd8QUs=' xml:lang='en-US' type='result' to='jvbbrewery@internal.machine/focus'>
machine # [   43.666734] prosody[1097]: internal.machine:muc: jvb@auth.machine/YFpi3EwZyPs- sent private iq stanza to jvbbrewery@internal.machine/focus (focus@auth.machine/focus)
machine # [   43.668633] prosody[1097]: c2s2736ef0: Sending[c2s]: <iq type='result' to='focus@auth.machine/focus' id='SPSYT-13' from='jvbbrewery@internal.machine/machine' xml:lang='en-US'>
machine # [   47.156676] prosody[1097]: c2s2dd79f0: Received[c2s]: <presence to='jvbbrewery@internal.machine/machine' xml:lang='en-US'>
machine # [   47.158145] prosody[1097]: internal.machine:muc: presence update for jvbbrewery@internal.machine/machine from session jvb@auth.machine/YFpi3EwZyPs-
machine # [   47.165716] prosody[1097]: c2s2736ef0: Sending[c2s]: <presence xml:lang='en-US' from='jvbbrewery@internal.machine/machine' to='focus@auth.machine/focus'>
machine # [   47.170140] prosody[1097]: c2s2dd79f0: Sending[c2s]: <presence xml:lang='en-US' from='jvbbrewery@internal.machine/machine' to='jvb@auth.machine/YFpi3EwZyPs-'>
cleanup
kill machine (pid 8)
machine # qemu-kvm: terminating on signal 15 from pid 6 (/nix/store/fkcl1wzq3106qqgl84bhgk1lp56q6bzg-python3-3.10.7/bin/python3.10)
(finished: cleanup, in 0.31 seconds)
Traceback (most recent call last):
  File "/nix/store/j3p569a2x2gfph2l5wcpqgvb7y9akfyw-nixos-test-driver-1.1/bin/.nixos-test-driver-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/j3p569a2x2gfph2l5wcpqgvb7y9akfyw-nixos-test-driver-1.1/lib/python3.10/site-packages/test_driver/__init__.py", line 114, in main
    driver.run_tests()
  File "/nix/store/j3p569a2x2gfph2l5wcpqgvb7y9akfyw-nixos-test-driver-1.1/lib/python3.10/site-packages/test_driver/driver.py", line 146, in run_tests
    self.test_script()
  File "/nix/store/j3p569a2x2gfph2l5wcpqgvb7y9akfyw-nixos-test-driver-1.1/lib/python3.10/site-packages/test_driver/driver.py", line 142, in test_script
    exec(self.tests, symbols, None)
  File "<string>", line 7, in <module>
  File "/nix/store/j3p569a2x2gfph2l5wcpqgvb7y9akfyw-nixos-test-driver-1.1/lib/python3.10/site-packages/test_driver/machine.py", line 631, in wait_until_succeeds
    retry(check_success, timeout)
  File "/nix/store/j3p569a2x2gfph2l5wcpqgvb7y9akfyw-nixos-test-driver-1.1/lib/python3.10/site-packages/test_driver/machine.py", line 131, in retry
    raise Exception(f"action timed out after {timeout} seconds")
Exception: action timed out after 30 seconds
kill vlan (pid 7)
```
does it pass for you?

cleeyv committed:
nixos/tests/jibri: remove grep for obsolete videobridge health check

As already fixed earlier this year in the jitsi-meet test by @ajs124
https://github.com/NixOS/nixpkgs/commit/729764a32d6569e692e44620bd98b552a9f72c33

cleeyv commented:
I saw the same test failure that you posted, and after some debugging realized that you had encountered the same problem in the jitsi-meet test earlier this year and fixed it by removing the log grep for an obsolete videobridge health check. I have made the same fix here and the test now completes successfully.

cleeyv commented:
I saw the same test failure that you posted, and after some debugging realized that you had encountered the same problem in the jitsi-meet test earlier this year and fixed it by removing the log grep for an obsolete videobridge health check. I have made the same fix here and the test now completes successfully.
